src/fontra/views/editor/editor.js

```diff
@@ -1574,6 +1583,46 @@ export class EditorController {
     this.sceneController.autoViewBox = false;
   }
 
+  toggleFullscreen() {
+    if (document.fullscreenElement) {
+      if (document.exitFullscreen) {
+        document.exitFullscreen();
+      }
+    } else {
+      const element = document.documentElement;
+      if (element.requestFullscreen) {
```

Comment:
We can only get here is fullscreen mode is enabled. So, like my comment above, is it possible for `element.requestFullscreen` to not exist here? If so: I would write ` element.requestFullscreen?.()` and leave out the `if` condition.